src/orders.rs

```diff
@@ -110,6 +110,11 @@ impl<Ms, Mdl, ElC: View<Ms>, GMs> OrdersContainer<Ms, Mdl, ElC, GMs> {
             app,
         }
     }
+
+    pub(crate) fn append_from(&mut self, mut other: Self) {
+        self.should_render = other.should_render;
+        self.effects.append(&mut other.effects);
+    }
```

Comment:
Could we rename it to something like `merge`? 
`append_from` implies that `orders` is only a container like `Vector` and we assume the user knows that it contains `effects`.

`merge` examples: [Lodash](https://lodash.com/docs/4.17.15#merge) or [Elixir](https://hexdocs.pm/elixir/Map.html#merge/2)

Reply:
It's supposed to be internal (`pub(crate)`) but I don't mind.

Reply:
> It's supposed to be internal (pub(crate)) but I don't mind.

Yeah, but I think you'll thank your younger self as soon as you get back to this code :)

You'll hate me, but can you change it to only `merge` and not `merge_with`?
Suffix `*_with` implies that the function expects a merging function.
Examples: 
  - [Vec::resize_with](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.resize_with)
  - [_.mergeWith](https://lodash.com/docs/4.17.15#mergeWith)
  - [Option::get_or_insert_with](https://doc.rust-lang.org/core/option/enum.Option.html#method.get_or_insert_with)

Reply:
Hahaha, my younger self.

Ah, yeah. Didn't realize about the `with`.